docs/quick-start/write-data.txt

```diff
@@ -32,6 +32,17 @@ Write Data to MongoDB
 
    .. step:: Add an API route that calls the controller function
 
+      Generate an API route file by running the following command:
+
+      .. code-block:: bash
+
+         php artisan install:api
+
+      .. tip::
+
+         Skip this step if you are using Laravel 10.x since the file that
```

Comment:
S: I'd switch to "because" (style guide doesn't like "since")
```suggestion
         Skip this step if you are using Laravel 10.x because the file that
```

Reply:
Good catch! Will update.